packages/console/README.md

```diff
@@ -0,0 +1 @@
+# @patternfly-react/console
```

Comment:
still need some notes here? i think we should mention the scripts added above and also mention this is an interim solution until we are on the lerna path...